admin/class-convertkit-wp-list-table.php

```diff
@@ -225,4 +248,77 @@ function ( $a, $b ) {
 
 	}
 
+	/**
+	 * Returns whether a search has been performed on the table.
+	 *
+	 * @since   3.0.0
+	 *
+	 * @return  bool    Search has been performed.
+	 */
+	public function is_search() {
+
+		return filter_has_var( INPUT_GET, 's' );
```

Comment:
Why is this `filter_has_var( INPUT_GET, 's' )` when you also have `array_key_exists( 's', $_REQUEST )`?

Reply:
When search is enabled for this, we'll always need to call `is_search`:
- `array_key_exists( 's', $_REQUEST )` here will rightly fail for WordPress' coding standards due to there not always being a nonce.
- `filter_has_var( INPUT_GET, 's' )` doesn't require a nonce for WordPress' coding standards.

`array_key_exists( 's', $_REQUEST )` can be used in `get_search`, as we only want to get the search parameter entered if there's a nonce, therefore we're not failing coding standards.

Reply:
Is this right: we want a way to check if this is a search without checking a nonce, so we use filter_has_var( INPUT_GET, 's' ), then when we actually want to see what the search is, we call array_key_exists( 's', $_REQUEST ) which does require a nonce check?